pimcore/lib/Pimcore/WorkflowManagement/Workflow/Manager.php

```diff
@@ -678,7 +703,7 @@ private function unregisterActionEvents()
 
     /**
      * Returns whether or not an element has a workflow
-     * @param Asset|ConcreteObject|Document $element
+     * @param AbstractElement $element
```

Comment:
More specific type hints are possible to get better IDE autocompletion when the actual type is too generic, so something like `AbstractElement|Asset|ConcreteObject|Document` would be OK in my mind.

As this isn't really of use here, I'm fine with changing it to `AbstractElement` only.